utils/errors/notifier/utils.go

```diff
@@ -0,0 +1,36 @@
+package notifier
+
+import (
+	"github.com/carousell/Orion/utils/log/loggers"
+	"github.com/getsentry/raven-go"
+)
+
+func (s severity) String() string {
+	return string(s)
+}
+
+func (s severity) RavenSeverity() raven.Severity {
+	return raven.Severity(s)
+}
+
+func (s severity) LoggerLevel() loggers.Level {
+	switch s {
+	case warningSeverity:
+		return loggers.WarnLevel
+	case infoSeverity:
+		return loggers.InfoLevel
+	case debugSeverity:
+		return loggers.DebugLevel
+	default:
+		return loggers.ErrorLevel
+	}
```

Comment:
Add

```
case errorSeverity:
  return loggers.ErrorLevel
```

instead of relying on the default case to handle the error severity.

So that if someone changes the default case in the future for some reason, they won't accidentally affect calls that are using `errorSeverity`

Reply:
fair point, updated.